File: Proyecto Mis Finanzas Java/src/gestorAplicación/interno/Suscripcion.java
package gestorAplicación.interno;

import java.util.ArrayList;

public enum Suscripcion{
	DIAMANTE(13, 0.80F, 4, 2f),
	ORO(10, 0.60F, 3, 1.5f),
	PLATA(7, 0.40F, 2, 1f),
	BRONCE(4, 0.20F, 2, 0.5f);
	
	//Atributos
	private final int limiteCuentas;
	private final float probabilidad_Inversion;
	private final int maxDeudas;
	private final float porcentajePrestamo;
	
	//Constructor
	Suscripcion(int limiteCuentas, float probabilidad_Inversion, int maxDeudas, float porcentajePrestamo){
		this.limiteCuentas = limiteCuentas;
		this.probabilidad_Inversion = probabilidad_Inversion;
		this.maxDeudas = maxDeudas;
		this.porcentajePrestamo = porcentajePrestamo;
	}
	
	//Métodos
	public int getLimiteCuentas() { return limiteCuentas; }
	
	public float getProbabilidad_Inversion() { return probabilidad_Inversion; }

	public int getMaxDeudas(){
		return maxDeudas;
	}

	public float getPorcentajePrestamo(){
		return porcentajePrestamo;
	}
}

File: Proyecto Mis Finanzas Python/src/gestorAplicación/interno/suscripcion.py
from enum import Enum
class Suscripcion(Enum):
    #Constantes
    DIAMANTE = (13, 0.80, 4, 2.0)
    ORO = (10, 0.60, 3, 1.5)
    PLATA = (7, 0.40, 2, 1)
    BRONCE = (4, 0.20, 2, 0.5)

    #Constructor
    def __init__(self, _LIMITECUENTAS, _PROBABILIDADINVERSION, _MAXDEUDAS, _PORCENTAJEPRESTAMO) -> None:
        #Atributos de instancia
        self._LIMITECUENTAS = _LIMITECUENTAS
        self._PROBABILIDADINVERSION = _PROBABILIDADINVERSION
        self._MAXDEUDAS = _MAXDEUDAS
        self._PORCENTAJEPRESTAMO = _PORCENTAJEPRESTAMO
    
    #Métodos de instancia
    def getLimiteCuentas(self) -> int:
        return self._LIMITECUENTAS
    def getProbabilidadInversion(self) -> float:
        return self._PROBABILIDADINVERSION
    def getMaxDeudas(self) -> int:
        return self._MAXDEUDAS
    def getPorcentajePrestamo(self) -> float:
        return self._PORCENTAJEPRESTAMO

    #Métodos estáticos
    @staticmethod
    def getNivelesSuscripcion() -> list:
        return([Suscripcion.DIAMANTE, Suscripcion.ORO, Suscripcion.PLATA, Suscripcion.BRONCE])
